core/src/main/java/org/apache/iceberg/BaseContentScanTask.java

```diff
@@ -82,7 +83,7 @@ public long start() {
 
   @Override
   public long length() {
-    return file.fileSizeInBytes();
+    return ScanTaskUtil.contentSizeInBytes(file);
```

Comment:
This is probably correct. Is this for `PositionDeletesScanTask`? Do we need to update `start` if we update `length`?

Reply:
yes this is for `BasePositionDeletesScanTask`. It didn't seem like `start` would need to be updated. Did you have something in particular in mind with updating `start`?

Reply:
No particular use case right now. One may argue `start` for DVs should return `contentOffset` from the correctness point of view.